### PR TITLE
Create CDF Project Social Media Guide.md

### PR DESCRIPTION
**Pull request by zestyroxy**
Here is the guide we discussed to help the project post on social media (I added a table for the images at the bottom, otherwise they were too wide -- open to other ideas)

### CODE REVIEW

**Review by TracyRagan — COMMENTED** [review]
Looks fine to me. On the twitter topic, we may want to look elsewhere. X is getting a bid weird.

**Comment by zestyroxy** [comment]
> On the twitter topic, we may want to look elsewhere. X is getting a bid weird.

Yes, but I'm not sure where. Seems like everyone is a little bit everywhere now.

File: CDF Project Social Media Guide.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+### General
+* #ContinuousDelivery
+* #DevOps
+* #GitOps
+
+
+## Project Accounts
+
+### Twitter
```
Comment (afrittoli) [inline CDF Project Social Media Guide.md:30]: Maybe this could be X (Twitter) to reflect the new name

**Review by afrittoli — APPROVED** [review]
Thanks @zestyroxy - this looks great.

I agree that we may need to add presence to X (Twitter) alternatives, but I'd be happy to add those after we merge this.

I think a popular alternative is mastodon https://mastodon.social/explore but in any case the first step might be to create a CDF account there. 

Is there any tool available to manage multiple of this accounts at once?

File: CDF Project Social Media Guide.md (hunk 1 above)
Comment (zestyroxy) [reply CDF Project Social Media Guide.md:30]: We can do that but I was resisting the silliness of the new name :D

**Comment by zestyroxy** [comment]
We already have a Mastadon account but I haven't use it yet. https://social.lfx.dev/@CDeliveryFdn

Social media at the LF are managed via Hubspot and Mastadon isn't supported, but I can start posting there.

**Pull request merged**